contrib/polygon/backfill/backfiller/backfiller.go

```diff
@@ -74,79 +63,59 @@ func main() {
 		}
 	}()
 
-	initWriter()
-
-	if apiKey == "" {
-		log.Fatal("[polygon] api key is required")
-	}
-
-	api.SetAPIKey(apiKey)
-
-	start, err := time.Parse(format, from)
-	if err != nil {
-		log.Fatal("[polygon] failed to parse from timestamp (%v)", err)
-	}
-
-	end, err := time.Parse(format, to)
-	if err != nil {
-		log.Fatal("[polygon] failed to parse to timestamp (%v)", err)
-	}
-
 	var symbolList []string
-	log.Info("[polygon] listing symbols for pattern: %v", symbols)
-	pattern := glob.MustCompile(symbols)
-	resp, err := api.ListTickers()
-	if err != nil {
-		log.Fatal("[polygon] failed to list symbols (%v)", err)
-	}
-	log.Info("[polygon] %v symbols available", len(resp.Tickers))
-	symbolList = make([]string, 1)
-	for _, s := range resp.Tickers {
-		if pattern.Match(s.Ticker) {
+	if len(config.Symbols) > 0 {
+		symbolList = config.Symbols
+	} else {
+		log.Info("[polygon] fetching available symbols")
+		resp, err := api.ListTickers()
+		if err != nil {
+			log.Fatal("[polygon] failed to list symbols (%v)", err)
+		}
+
+		symbolList := make([]string, 1)
+		for _, s := range resp.Tickers {
 			symbolList = append(symbolList, s.Ticker)
 		}
```

Comment:
[nit] slightly faster as there will be no new allocation
```suggestion
		symbolList := make([]string, len(resp.Tickers))
		for i, s := range resp.Tickers {
			symbolList[i] = s.Ticker
		}
```

Reply:
Ah nice, thanks!